smallrye-reactive-messaging-in-memory/src/test/java/io/smallrye/reactive/messaging/connectors/WeldTestBase.java

```diff
@@ -0,0 +1,154 @@
+package io.smallrye.reactive.messaging.connectors;
```

Comment:
Is this class somewhere it doesn't get pulled in as a dependency? Do we need a common test dependency?

Reply:
At some point, we may want to externalize some of the duplicated test classes somewhere. I also need to check if I cannot use Weld unit directly.